packages/amplify-codegen/src/utils/getProjectRoot.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Find the project root.
+ * @param {!import('@aws-amplify/amplify-cli-core').$TSContext} context the amplify runtime context
+ * @returns {!string} the project root, or cwd
+ */
+const getProjectRoot = (context) => {
```

Comment:
Moved out of `commands/model.js` since it's reused in the `getOutputDirParam` utility.